src/test/java/mat/server/CqlAttributesRemoteCallTest.java

```diff
@@ -0,0 +1,50 @@
+package mat.server;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import mat.client.shared.CQLConstantContainer;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.client.RestTemplate;
+
+import java.io.File;
+import java.net.URL;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+@ExtendWith(MockitoExtension.class)
+class CqlAttributesRemoteCallTest {
+
+    @Value("https://localhost:9090")
```

Comment:
Value annotation is not needed here. It's not used

Reply:
Cleared